.github/workflows/docs.yml

```
@@ -3,6 +3,8 @@ on:
   push:
     branches:
       - master
+    paths:
+      - 'docs/**'
     # Build pull requests
     pull_request:
```

This line is at the wrong indentation (pull_request is another event type like push). If your intent is to run this for commits on master that touch docs AND for pull requests that touch docs, then I think you want:
```suggestion
  # Build pull requests
  pull_request:
    paths:
      - 'docs/**'
```

---

Yes, that was my intent. Thank you! I've committed your suggestion.